REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.4
-Compat 0.7.14
+Compat 0.8.9
```

Comment:
Well, I don't really care what it is going to be right now. So if I can't put a place holder like `0.8.8+` and I can't just put what is currently the next version should I just leave the version requirement unchanged?